mount/mount.go

```diff
@@ -16,6 +16,12 @@
 
 package mount
 
+import (
+	"fmt"
+
+	"github.com/containerd/continuity/fs"
```

Comment:
I'll probably blame myself if I don't comment; was looking if this would be an issue to add as new dependency for this package (and if another approach would work). Looks like continuity was already a dependency (but for testing only). I didn't see a good alternative immediately though (resolving the path before calling the `mount` package didn't seem like an immediate option, so "probably" fine.